src/components/MultipleCurrencyLogo/index.tsx

```diff
@@ -32,13 +32,14 @@ export default function MultipleCurrencyLogo({
   currencies,
   size = 16,
   margin = false,
-  separation
+  separation,
+  ...rest
 }: MultipleCurrencyLogoProps) {
   const currenciesQty = currencies.length
   const logosSeparation = separation ?? 10
 
   return (
-    <Wrapper sizeraw={size} margin={margin}>
+    <Wrapper sizeraw={size} margin={margin} {...rest}>
```

Comment:
If `...rest` is to support the `styled(MultipleCurrencyLogo)` below, I think instead of doing `...rest`, you can just do `className` and pass `className` in to `<Wrapper ... >`

Reply:
Indeed 👌🏻